examples/dns-interceptor/src/pipeline.rs

```diff
@@ -17,43 +18,53 @@ impl route_rs_runtime::pipeline::Runner for Pipeline {
         input_channel: crossbeam::Receiver<Self::Input>,
         output_channel: crossbeam::Sender<Self::Output>,
     ) {
+        let mut all_runnables: Vec<TokioRunnable> = vec![];
+
         let elem_1_setinterfacebydestination = SetInterfaceByDestination::new();
         let elem_2_classifydns = ClassifyDNS::new();
         let elem_3_localdnsinterceptor = LocalDNSInterceptor::new();
 
         let link_1 = InputChannelLink::new(input_channel);
```

Comment:
It would be nice if the links had more descriptive names, but if this is a code generation limitation then it's fine.

Reply:
I think it's sort of a question of how we expect people to be working with the pipeline. Right now the Links are kind of invisible, and what you see is the Processors and such. I do think we could do more things to make the file navigable in big projects.